src/test/java/org/kainos/ea/service/ResponsibilityServiceUnitTest.java

```diff
@@ -0,0 +1,69 @@
+package org.kainos.ea.service;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.kainos.ea.api.ResponsibilityService;
+import org.kainos.ea.cli.Responsibility;
+import org.kainos.ea.db.DatabaseConnector;
+import org.kainos.ea.db.ResponsibilityDao;
+import org.kainos.ea.exception.FailedToGetResponsibilitiesException;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@ExtendWith(MockitoExtension.class)
+public class ResponsibilityServiceUnitTest {
+
+    ResponsibilityDao responsibilityDao = Mockito.mock(ResponsibilityDao.class);
+    DatabaseConnector databaseConnector = Mockito.mock(DatabaseConnector.class);
+
+    ResponsibilityService responsibilityService = new ResponsibilityService(responsibilityDao, databaseConnector);
+    Connection connection;
+
+    @Test
```

Comment:
add tests for getResponsibilitiesForJob method

Reply:
getAllResponsibilities test have been removed, as the method has been removed from the Dao. Tests added for getResponsibilitiesForJob method